lib/mutant/ast/meta.rb

```diff
@@ -19,12 +19,11 @@ class Send
         INDEX_ASSIGNMENT_SELECTOR            = :[]=
         ATTRIBUTE_ASSIGNMENT_SELECTOR_SUFFIX = '='.freeze
 
-        # Return arguments
+        # The arguments of node
```

Comment:
For my limited English this carries the same information. But I'll simply use your suggestion as its likely your suggestion is what I actually meant.